Cargo.toml

```diff
@@ -42,11 +42,11 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 [dependencies]
 cosmwasm-std = { version = "1.0.0-beta" }
 cosmwasm-storage = { version = "1.0.0-beta" }
-cw-storage-plus = "0.10"
-cw2 = "0.10"
-schemars = "0.8.3"
-serde = { version = "1.0.127", default-features = false, features = ["derive"] }
-thiserror = { version = "1.0.26" }
+cw-storage-plus = "0.11"
+cw2 = "0.11"
+schemars = "0.8"
+serde = { version = "1.0.12", default-features = false, features = ["derive"] }
```

Comment:
Why lowering the version of serde and thiserror?

I think we bumped them earlier to ensure we got the advantages.
Is there anyone that requires such old versions?

Reply:
Same as https://github.com/CosmWasm/cw-tokens/pull/5#discussion_r776233594.
```
T/cw-template.F7vJMjN1w/testgen-local  update-0.11 ✗                                                                                                ◒  
▶ cargo tree | ag thiserror
│   │   │   └── thiserror v1.0.26
│   │   │       └── thiserror-impl v1.0.26 (proc-macro)
│   │   └── thiserror v1.0.26 (*)
│   ├── thiserror v1.0.26 (*)
└── thiserror v1.0.26 (*)
```

Reply:
Yes, but this means it accepts any serde from 1.0.12 on, not 1.0.100 on.

I remember some issues with very old serde (like 2 years ago), so I would rather provide a bit newer minimum version, like 1.0.83 at least